solutions/observability/streams/management/extract/manual-pipeline-configuration.md

```diff
@@ -5,6 +5,10 @@ applies_to:
 ---
 # Manual pipeline configuration [streams-manual-pipeline-configuration]
 
+:::{note}
+The manual pipeline configuration processor is only available on [classic streams](../../streams.md#streams-classic-vs-wired).
+:::
+
```

Comment:
Commenting here since I can't comment below. [This reference](https://github.com/elastic/docs-content/pull/3918/files#diff-393237e93e1e59df09a05259bdab876e9a6b9c2539d176c842f60d0bf9dfb69fL14) isn't rendering correctly in the preview. I haven't verified the path yet but did notice that you used square brackets around `elasticsearch://reference/enrich-processor.md` instead of parentheses (`()`). Changing the brackets to parentheses might do the trick.